library/ix-dev/community/node-red/questions.yaml

```diff
@@ -21,6 +21,15 @@ portals:
     path: "$kubernetes-resource_configmap_portal_path"
 
 questions:
+  - variable: TZ
+    group: Node-RED Configuration
+    label: Timezone
+    schema:
+      type: string
+      default: America/Los_Angeles
```

Comment:
Let's be consistent with the rest and default to Etc/UTC

Reply:
OK - Fixed